err_max : 0.15
if_debug : 1
zero_plane_height : 0.2
height_threshold : 0.05
min_range : 0.1
max_range : 30.0
num_bins : 360

R_outside : 2.0
R_inside : 1.0
p_R : 1.0 
p_r : 2.0
p_xy : 0.5
vel_collision_max : 1.0
vel_track_max : 1.0
vel_sp_max : 1.5
